applications/HDF5Application/python_scripts/core/__init__.py

```diff
@@ -2,11 +2,17 @@
 
 license: HDF5Application/license.txt
 '''
+
+
+__all__ = ["Factory"]
```

Comment:
what does this do?

Reply:
Declares the names to import when doing `from core import *`. Other names can still be accessed with dot operator.